modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,296 @@
+====
+
+. Create the config map in the management cluster:
++
+[source, terminal]
+----   
+$ oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-1.yaml
+----
+
+. Reference the config map in the node pool's `spec.tunedConfig` field, either by editing a node pool or creating one. In this example, assume that you have only one node pool, `nodepool-1`, which contains 2 nodes.
+
++
+[source,yaml]
+----
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+      ...
+      name: nodepool-1
+      namespace: clusters
+    ...
+    spec:
+      ...
+      tunedConfig:
+      - name: tuned-1
+    status:
+    ...
+----
++
+[NOTE]
+====
+You can reference the same config map in multiple node pools. In hosted control planes, the Node Tuning Operator appends a hash of the node pool name and namespace to the name of the tuned custom resources (tuneds) to distinguish them. Outside of this case, do not create multiple tuned profiles of the same name in different tuneds for the same hosted cluster.
+====
+
+.Verification
+
+Now that you have created the config map that contains a tuned manifest and referenced it in a node pool, the Node Tuning Operator syncs the tuned objects into the hosted cluster. You can verify which tuneds are defined and which profiles are set for each node.
+
+. List the tuned objects in the hosted cluster:
++
+[source,terminal]
+----
+    oc --kubeconfig="$HC_KUBECONFIG" get Tuneds -n openshift-cluster-node-tuning-operator
+----
+
++
+.Example output
+[source,terminal]
+----
+    NAME       AGE
+    default    7m36s
+    rendered   7m36s
+    tuned-1    65s
+----
+   
+. List the profiles in the hosted cluster:
++
+[source,terminal]
+----
+    oc --kubeconfig="$HC_KUBECONFIG" get Profiles -n openshift-cluster-node-tuning-operator
+----
+
++
+.Example output
+[source,terminal]
+----
+    NAME                           TUNED            APPLIED   DEGRADED   AGE
+    nodepool-1-worker-1            tuned-1-profile  True      False      7m43s
+    nodepool-1-worker-2            tuned-1-profile  True      False      7m14s
+----
+
++
+[NOTE]
+====
+If no custom profiles are created, the `openshift-node` profile is applied by default.
+====
+
+
+. To confirm that the tuning was applied correctly, start a debug shell on a node and check the sysctl values:
++
+[source,terminal]
+----
+    oc --kubeconfig="$HC_KUBECONFIG" debug node/nodepool-1-worker-1 -- chroot /host sysctl vm.dirty_ratio
+----
++
+.Example output
+[source,terminal]
+----
+    vm.dirty_ratio = 55
+----
+
+[id="advanced-node-tuning-hosted-cluster_{context}"]
```

Comment:
One procedure per module where possible, according to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures). I recommend breaking this out into a second module.